airbyte-webapp/src/views/Connector/ConnectorForm/components/Property/LabelInfo.module.scss

```
@@ -18,6 +18,17 @@
   margin-top: 5px;
 }
 
+.optionsHeader {
+  margin-top: 0;
+  margin-bottom: 5px;
```

```suggestion
  margin-bottom: variables.$spacing-sm;
```

nit: we should replace the px values in this file with our `_variables.scss` spacing values: https://github.com/airbytehq/airbyte/blob/5da3a7c2a80814bef8cc10e18f4faad93229d3da/airbyte-webapp/src/scss/_variables.scss#L16-L20

For ones that don't have a matching value (e.g. 25px), we should try out the closest value that matches, e.g. `spacing-xl` and see if that looks fine.